worlds/ladx/ItemIconGuessing.py

```diff
@@ -96,6 +97,7 @@
     'BLAST': 'BOMB',
     'BOMBCHU': 'BOMB',
     'FIRECRACKER': 'BOMB',
+    'SURGE': 'BOMB',
```

Comment:
This one has way fewer uses, though it's still a bit questionable -- Fire Surge and Acid Surge spells in DS3, Chrono Surge in SC2